src/devices/StUsb4500/Objects/ObjectBase.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.CompilerServices;
+
+namespace Iot.Device.StUsb4500.Objects
+{
+    /// <summary>
+    /// Base class for shared object logic.
+    /// </summary>
+    public abstract class ObjectBase
```

Comment:
these should be internal static helper class and not public abstract base class

Reply:
@krwq what would be the benefit of using a helper instead of a base class?